definitions/npm/react-dnd_v2.x.x/flow_>=v0.23.x/react-dnd_v2.x.x.js

```diff
@@ -87,14 +89,14 @@ type DragPreviewOptions = {
 }
 
 type ConnectDragSource = (
-  elementOrNode: React$Element<any>
+  elementOrNode: ElementOrNode
   options?: DragSourceOptions
-) => React$Element<any>;
+) => ElementOrNode;
```

Comment:
Is there any way to express that if you pass element, you get an element, but if you pass a node, you get a node?